### PR TITLE
Update README.md to use new repository name

### PR DESCRIPTION
**Pull request by jensfr**
**-What I did**
Rename all occurrences of the github repo name. Basically s/openshift\/kata-operator/sandboxed-containers/operator/g

**- How to verify it**
Read the document and search for typos or missed renames

**- Description for the changelog**
Use new repository name in README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
-## :information_source: If you are using OCP 4.6 please follow this [README](https://github.com/openshift/kata-operator/blob/release-4.6/README.md)
+## :information_source: If you are using OCP 4.7 please follow this [README](https://github.com/openshift/sandboxed-containers-operator/blob/release-4.7/README.md)
+## :information_source: If you are using OCP 4.6 please follow this [README](https://github.com/openshift/sandboxed-containers-operator/blob/release-4.6/README.md)
 
 # Kata Operator
```
Comment (bpradipt) [inline README.md:4]: Should reference to "Kata Operator" in the above section and following instructions be changed as well ?

File: README.md [hunk 2]
```diff
@@ -15,7 +16,7 @@ An operator to perform lifecycle management (install/upgrade/uninstall) of [Kata
 
 
    ```
-   git clone https://github.com/openshift/kata-operator 
+   git clone https://github.com/openshift/sandboxed-containers-operator 
    git checkout -b master --track origin/master
    ```
 3. Install the Kata Operator on the cluster,
```
Comment (bpradipt) [inline README.md:22]: Same comment as above. Kata Operator => Sandboxed Containers Operator ?

File: README.md (hunk 1 above)
Comment (jensfr) [reply README.md]: At first I only wanted to change URLs, but you're right why not change the name of the operator in the text as well. I changed it from "Kata Operator" to "OpenShift sandboxed containers operator" now.

**Review by bpradipt — APPROVED** [review]
/lgtm
Thanks @jensfr

**Review by cpmeadors — COMMENTED** [review]
All instances of openshift/kata-operator have been replaced.  I am wondering if the "sandboxed containers operator" should be capitalized, i.e. "Sandboxed Containers Operator" like "Kata Operator" was.  That was not part of this PR but the changes are here.

**Comment by jensfr** [comment]
@cpmeadors I made it lower case on purpose after I looked at how @zanetworker usually writes it.

**Review by cpmeadors — APPROVED** [review]
got confirmation capitalization is correct

**Pull request merged**